requirements.txt

```diff
@@ -1,38 +1,37 @@
 boto==2.36.0
-setuptools==30.1.0
-psycopg2==2.7.3.2
 cryptography==2.1.2
+dictionaryutils
 envelopes==0.4
-flask-cors==1.9.0
-Flask-SQLAlchemy-Session==1.1
 Flask==0.10.1
+flask-cors==3.0.3
```

Comment:
what's up with all these upgrades?

Reply:
in the local environment, the versions of packages conflict to the one in authutils and peregrine. Just merge and test them to be compatible.